src/holofuel/components/CopyToClipboard/CopyToClipboard.js

```diff
@@ -13,7 +13,7 @@ export default function CopyToClipboard ({
 
   const handleCopyItem = async () => {
     const wasCopied = await copy(copyContent)
-    if (wasCopied === true) newMessage(messageText, 5000)
+    if (wasCopied === true) newMessage(messageText,  0)
```

Comment:
Why is this timeout changed?

Reply:
Good catch; this is def residue from testing... changing it back now.